gradle/dependencies.gradle

```diff
@@ -25,26 +25,37 @@ ext {
   defaultScalaVersions = [ '2.11', '2.12' ]
   // Available if -PscalaVersion is used. This is useful when we want to support a Scala version that has
   // a higher minimum Java requirement than Kafka. This was previously the case for Scala 2.12 and Java 7.
-  availableScalaVersions = [ '2.11', '2.12' ]
+  availableScalaVersions = [ '2.11', '2.12', '2.13' ]
 }
 
 // Add Scala version
 def defaultScala211Version = '2.11.12'
 def defaultScala212Version = '2.12.8'
+def availableScala213Version = '2.13.0-M5' // keep in mind to align milestone/rc Scala 2.13.0-XY version with 'scovarage' version below (and vice versa)
```

Comment:
I think I'd just call this `defaultScala213Version`. It's the default for that release series. The fact that it's not available is orthogonal.

Reply:
Done